src/main/java/wooteco/subway/dao/station/JdbcStationDao.java

```diff
@@ -0,0 +1,65 @@
+package wooteco.subway.dao.station;
+
+import java.sql.PreparedStatement;
+import java.util.List;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+import wooteco.subway.domain.station.Station;
+
+@Repository
+public class JdbcStationDao implements StationDao {
+
+    private final RowMapper<Station> stationRowMapper = (resultSet, rowNum) -> {
+        return new Station(
+                resultSet.getLong("id"),
+                resultSet.getString("name")
+        );
+    };
+
+    private final JdbcTemplate jdbcTemplate;
+
+    public JdbcStationDao(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    @Override
+    public long save(Station station) {
+        final String sql = "insert into STATION (name) values (?)";
+
+        KeyHolder keyHolder = new GeneratedKeyHolder();
+        jdbcTemplate.update(connection -> {
+            PreparedStatement preparedStatement = connection.prepareStatement(sql, new String[]{"id"});
+            preparedStatement.setString(1, station.getName());
+            return preparedStatement;
+        }, keyHolder);
+
+        return keyHolder.getKey().longValue();
+    }
+
+    @Override
+    public boolean existStationById(Long id) {
+        final String sql = "select exists (select * from STATION where id = ?)";
+        return jdbcTemplate.queryForObject(sql, Boolean.class, id);
+    }
+
+    @Override
+    public boolean existStationByName(String name) {
+        final String sql = "select exists (select * from STATION where name = ?)";
+        return jdbcTemplate.queryForObject(sql, Boolean.class, name);
```

Comment:
```suggestion
Boolean.TRUE.equals(jdbcTemplate.queryForObject(sql, Boolean.class, name));
```

NPE를 발생시킬수 있어 이렇게 많이 작성하는 편입니다 :)